{"uuid": "bf344d80-3bb0-4c2e-aa18-b7a32138cce7", "children": ["e63a3d59-f03a-419c-8728-f21f0e19ce96"], "befores": [{"name": "feed_explorer_agent", "status": "broken", "statusDetails": {"message": "KeyError: 'llama3.2'\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/OA/OA/tests/agent_trajectory/feed_explore.py\", line 13, in feed_explorer_agent\n    llm = get_available_providers()[model]\n          ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^\n"}, "start": 1730427896711, "stop": 1730427899097}], "start": 1730427896711, "stop": 1730427903091}